app/src/main/java/org/jellyfin/androidtv/di/SearchModule.kt

```diff
@@ -0,0 +1,34 @@
+package org.jellyfin.androidtv.di
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.core.app.NotificationManagerCompat
+import org.jellyfin.androidtv.BuildConfig
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.data.service.BackgroundService
+import org.jellyfin.androidtv.preference.UserPreferences
+import org.jellyfin.androidtv.ui.browsing.MainActivity
+import org.jellyfin.androidtv.ui.playback.GarbagePlaybackLauncher
+import org.jellyfin.androidtv.ui.playback.LegacyMediaManager
+import org.jellyfin.androidtv.ui.playback.RewritePlaybackLauncher
+import org.jellyfin.androidtv.ui.playback.VideoQueueManager
+import org.jellyfin.androidtv.ui.playback.rewrite.RewriteMediaManager
+import org.jellyfin.androidtv.ui.search.SearchFragmentDelegate
+import org.jellyfin.playback.core.mediasession.MediaSessionOptions
+import org.jellyfin.playback.core.mediasession.mediaSessionPlugin
+import org.jellyfin.playback.core.playbackManager
+import org.jellyfin.playback.exoplayer.exoPlayerPlugin
+import org.jellyfin.playback.jellyfin.jellyfinPlugin
+import org.koin.core.scope.Scope
+import org.koin.dsl.module
+import org.jellyfin.androidtv.ui.playback.PlaybackManager as LegacyPlaybackManager
+
+val searchModule = module {
+	factory { (context: Context) ->
+		SearchFragmentDelegate(context, get())
```

Comment:
Let's put this in the app module for now, adding a module for a single instance doesn't seem worth it.